model/clusters_mgmt/v1/provision_shard_type.model

```diff
@@ -31,4 +31,7 @@ class ProvisionShard {
 
 	// Contains the GCP base domain
 	GCPBaseDomain String
+
+	// Contains the cloud-provider region in which the provisioner spins up the Hypershift cluster
```

Comment:
I'd leave the "Hypershift" part out of this comment because this also applies to Hive shards.